src/app/modules/diet-entity/components/diet-entity-summary/diet-entity-summary.component.ts

```diff
@@ -0,0 +1,38 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+import { FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'diet-entity-summary',
+  template: `
+      <div class="summary-wrapper" [formGroup]="summaryFormGroup">
+          <div class="summary">
+              <input type="number" class="summary-quantity-input" name="kcal" formControlName="kcal" value="234">
```

Comment:
value="234" to chyba przypadkowo zostawiłeś?

Reply:
yup